File: app/src/main/java/valcus/roundtable/gameLogic/Player.java
package valcus.roundtable.gameLogic;

/**
 * Created by Tim Shimp on 2/17/2018.
 */

public class Player {

    long playerId;
    int role;
    String displayName;

    public int getRole() {
        return role;
    }

    public void setRole(int role) {
        this.role = role;
    }

    public String getDisplayName() {
        return displayName;
    }

    public void setDisplayName(String displayName) {
        this.displayName = displayName;
    }
}

File: app/src/main/java/valcus/roundtable/gameLogic/Player.kt
package valcus.roundtable.gameLogic

import java.util.*

/**
 * Created by Tim Shimp on 2/17/2018.
 */

class Player {

    internal var playerId: UUID = UUID.randomUUID()
    var role: Int? = null
        set(value) {
            alignment = RoleInformation.getAlignment(value)
            field = value
        }
    var alignment: Int = 0
    var displayName: String? = null
}
